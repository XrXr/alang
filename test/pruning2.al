main :: proc() {
	a := 399
	b := a
	c := b
	print_int(a)
	print_int(b)
	print_int(c)
}
