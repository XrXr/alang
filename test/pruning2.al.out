399
399
399
